src/rpc/ws_client/mod.rs

```diff
@@ -108,271 +105,17 @@ impl HandleMessage for SubscriptionHandler {
 
 		match value["id"].as_str() {
 			Some(_idstr) => {},
```

Comment:
It should not happen, but maybe we want to log here, that we expected a subscription.

Reply:
Included a warn log

Reply:
Messi bokup!